ludwig/modules/embedding_modules.py

```diff
@@ -210,20 +174,26 @@ def forward(self, inputs, training=None, mask=None):
 
         return embedded
 
+    @property
+    def input_shape(self) -> torch.Size:
+        return torch.Size([1])
```

Comment:
this is an interesting case, I believe the same ambed is used also for seuqence/text features where the input size is not 1.

Reply:
@w4nderlust i looked at the imports of `sequence_encoders.py`, it looks like they use `EmbedSequence` instead of `Embed`.

However, it does look like `Embed` is used in category encoders, date encoders and h3 encoders. Do you know if the shape would change for any of them?

Reply:
Oh ok, it sounds good then. In date and H3 it is used to embed the single components, so it makes sense to have have size 1.